leaflet-ohm-timeslider.js

```diff
@@ -1052,14 +1137,17 @@ L.Control.OHMTimeSlider.Translations['es'] = {
     resetbutton_title: "Ir al inicio del alcance",
     autoplay_submit_text: "Aplicar",
     autoplay_submit_title: "Aplicar la configuración",
+    datepicker_month: "Mes",
+    datepicker_day: "Día",
+    datepicker_year: "Año",
+    datepicker_cebce: "a. C. o d. C.",
     datepicker_submit_text: "Aplicar fecha",
     datepicker_cancel_text: "Cancelar",
     datepicker_title: "Cambiar fecha",
     datepicker_format_text: "Formatos de fecha",
     datepicker_text: "Entra una nueva fecha para actualizar la ubicación del mango y los datos que se muestran.",
     months: ['Enero', 'Febrero', 'Marzo', 'Abril', 'Mayo', 'Junio', 'Julio', 'Agosto', 'Septiembre', 'Octubre', 'Noviembre', 'Diciembre'],
```

Comment:
We can also eliminate this array by getting the month names from `Intl.DateTimeFormat.formatToParts()` just like you’ve done for era names. We’re doing this in iD as of OpenHistoricalMap/iD#185:

https://github.com/OpenHistoricalMap/iD/blob/148c32ac3c2bde08e58fddec74900d7be170a4fa/modules/ui/fields/date.js#L78-L89

Let’s track this improvement as well in OpenHistoricalMap/issues#626.